ac_c.lua

```diff
@@ -72,11 +72,18 @@ Citizen.CreateThread(function()
 		if GetUsingnightvision() then
 			TriggerServerEvent("AntiCheese:Night")
 		end
-		
-		local playerPed = PlayerPedId()
-		if IsPedSittingInAnyVehicle(playerPed) and IsVehicleVisible(GetVehiclePedIsIn(playerPed, false)) then
+
+		if IsPedSittingInAnyVehicle(PlayerPedId()) and IsVehicleVisible(GetVehiclePedIsIn(PlayerPedId(), false)) then
 			TriggerServerEvent("AntiCheese:CarVisible")
 		end
+
+		if not IsEntityVisible(PlayerPedId()) then
+			TriggerServerEvent("AntiCheese:PlayerVisible")
+		end
+
+		if GetLocalPlayerAimState() ~= 3 and not IsPedInAnyVehicle(PlayerPedId()) then
```

Comment:
isnt this a setting in GTA? I dont think this is sane to list as a cheat, unless you cant actually change this setting in FiveM